cluster/examples/kubernetes/ceph/monitoring/prometheus-ceph-v14-rules.yaml

```diff
@@ -80,7 +80,7 @@ spec:
         severity_level: error
         storage_type: ceph
       expr: |
-        count(ceph_mon_quorum_status{job="rook-ceph-mgr"} == 1) <= ((count(ceph_mon_metadata{job="rook-ceph-mgr"}) % 2) + 1)
+        count(ceph_mon_quorum_status{job="rook-ceph-mgr"} == 1) <= (floor(count(ceph_mon_metadata{job="rook-ceph-mgr"}) / 2) + 1)
```

Comment:
@travisn @anmolsachan recently, we allowed having even mons too, is this something we need to handle here?

Reply:
This formula should also work for an even number of mons.